src/conn/conn_log.c

```diff
@@ -601,8 +601,14 @@ __log_file_server(void *arg)
         /*
          * If there is a log file to close, make sure any outstanding write operations have
          * completed, then fsync and close it.
+         *
+         * The read from log close file handle is ordered with respect to the log close lsn. We
+         * write to it in the order log close lsn, then log close file handle. As such we need to
+         * read from it the reverse order. We are not protected by an address dependency in this
+         * context.
          */
```

Comment:
```suggestion
         *
         * The read from the log close file handle is ordered with the read from the log close lsn. Writers
         * will set the log close lsn first and then the log close file handle, so we need to
         * read them in the reverse order to see a consistent state.
         */
```
Suggested reword. I've dropped the sentence about address dependency as I don't think developers need to be thinking about it in our current model.